bot/cogs/cyber.py

```diff
@@ -63,6 +63,10 @@ def __init__(self, bot: Bot):
             r"^.*\bwhat\b.*\belite\b.*\bemail\b.*$",
             re.IGNORECASE
         )
+        self.game_level_regex = re.compile(
+            r"^(le?ve?l)\b.*?(\d{1,2}).*?(\d{1,2})\b.*$",
```

Comment:
```suggestion
            r"^(le?ve?l)?\b.*?(\d{1,2}).*?(\d{1,2})\b.*$",
```

Reply:
Two numbers in the message shouldn't really trigger the command, do you agree?

Reply:
since this only runs in the relevant channels, it shouldnt be too much of an issue on misfires

however the regex can be edited to avoid them completely

Reply:
I've updated the regex so at least an L is required in the message to prevent missfires

So the following will work (any case):

level 125
level125
level 125
level 12 5
lvl 125
l12c5
l13.1

Reply:
thanks skiros!!